tests/fixture/tmpnet/network.go

```diff
@@ -957,47 +954,60 @@ const invalidRPCVersion = 0
 
 // checkVMBinaries checks that VM binaries for the given subnets exist and optionally checks that VM
 // binaries have the same rpcchainvm version as the indicated avalanchego binary.
-func checkVMBinaries(w io.Writer, subnets []*Subnet, avalanchegoPath string, pluginDir string) error {
+func checkVMBinaries(log logging.Logger, subnets []*Subnet, avalanchegoPath string, pluginDir string) error {
 	if len(subnets) == 0 {
 		return nil
 	}
 
-	expectedRPCVersion, err := getRPCVersion(avalanchegoPath, "--version-json")
+	avalanchegoRPCVersion, err := getRPCVersion(avalanchegoPath, "--version-json")
 	if err != nil {
-		// Only warn if the rpc version is not available to ensure backwards compatibility.
-		if _, err := fmt.Fprintf(w, "Warning: Unable to check rpcchainvm version for avalanchego: %v\n", err); err != nil {
-			return err
-		}
+		log.Warn("unable to check rpcchainvm version for avalanchego", zap.Error(err))
+		return nil
 	}
 
-	errs := []error{}
+	var incompatibleChains bool
```

Comment:
hm - I feel like the prior version which had descriptive error messages for each chain was better than this version which has a single generic error if any of them are invalid.

I suppose the thought was that we are logging the information now rather than including the information in the error?

Reply:
I'm fine with that rationale - this lgtm